src/colby/sp3000_color_by_numbers.cpp

```diff
@@ -397,11 +384,11 @@ sp3000_color_by_numbers::result sp3000_color_by_numbers::convert_impl (const cv:
 		immediate_image_factory () = delete;
 		immediate_image_factory (const cv::Mat & mat) noexcept : mat_(mat) {	}
 		virtual cv::Mat image () override {
-			return convert_lab_to_bgr(mat_);
+			return lab2bgr(mat_);
 		}
 	};
 	//	1. Convert the pixels to the CIELAB colour space
-	auto lab = convert_bgr_to_lab(src);
+	cv::Mat lab = bgr2lab(src);
```

Comment:
The fact that `auto` doesn't work (or doesn't seem to work) really bothers me especially since I strongly prefer [AAA style](https://herbsutter.com/2013/08/12/gotw-94-solution-aaa-style-almost-always-auto/)

Let me take a look at this later...